modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="configure-ocp-oauth-identity-provider_{context}"]
+= Configuring {ocp} Oauth server as an identity provider in {product-title}
+
+[role="_abstract"]
+Use the instructions in this section to integrate the built-in {ocp} OAuth server as an identity provider for {product-title}.
+
+.Prerequisites
+* You must have permissions to configure identity providers in {product-title}.
+* You must have an Okta app configured for {product-title}.
```

Comment:
Theres no need for an Okta App. I think you are referring to you OAuth App. Can you reference the link to configure that in OpenShift. You need the OAuth server setup and an identity provider configured. We'd then respect what was provided to us through that identity provider. See this section https://docs.openshift.com/container-platform/4.9/authentication/understanding-authentication.html